competencies/roles/DataScience.md

```diff
@@ -27,24 +27,24 @@ with [Developers](./Developer.md)/[DataEngineers](./DataEngineer.md) but also wi
             <strong>Business Alignment - "Good Questions"</strong>
         </td>
         <td><ul>
-            <li>Can assess questions based on whether they are precise enough to develop an analysis or model.</li>
-            <li>Can participate in and capture key requirements in team discussions of needed analysis. Translating concise business question to technical equivalent.</li>
-            <li>Effective at soliciting feedback on the effectiveness of analysis in solving problems.</li>
+            <li>Able to assess business questions and determine whether they are precise enough to develop an analysis or model.</li>
+            <li>Able to participate in team discussions regarding analysis and capture key requirements. Capable of translating business questions to concise technical asks.</li>
+            <li>Effective at soliciting feedback on the effectiveness of analyses.</li>
         </ul></td>
         <td><ul>
             <!--- Proficient  -->
-            <li>Active participation and outreach in explorating an ambiguous space, mapping to technical space, and communicating implications effectively.</li>
-            <li>Can participate in a business level objectives discussion and develop testable data hypotheses.</li>
-            <li>Systematically approach questions of effectiveness and identify adjacent opportunities or questions;
-                systematically check understanding of problem and relevance of solutions.</li>
+            <li>Active participation and outreach in explorating an ambiguous problem space, mapping to technical space, and  effectively communicating implications of analyses or technical methods.</li>
```

Comment:
Can you fix the typo here too please? `explorating`

Reply:
Explorating is an actual word, but can change it to exploring if it's more clear